lib/optimizely.rb

```diff
@@ -388,39 +388,43 @@ def get_feature_variable_for_type(feature_flag_key, variable_key, variable_type,
 
       variable_value = nil
       variable = @config.get_feature_variable(feature_flag, variable_key)
-      unless variable.nil?
-        variable_value = variable['defaultValue']
 
-        decision = @decision_service.get_variation_for_feature(feature_flag, user_id, attributes)
-        unless decision
-          @logger.log(Logger::INFO,
-            "User '#{user_id}' was not bucketed into any variation for feature flag '#{feature_flag_key}'. Returning the default variable value '#{variable_value}'.")
+      if variable.nil?
+        # Error message logged in ProjectConfig- get_feature_flag_from_key
+        return nil
+      else
```

Comment:
Please lint your code with Rubocop and resubmit. 
No need of else here, the above logic can be written in a single line
return nil if variable.nil?

Reply:
Done.